Quadriga/src/main/java/edu/asu/spring/quadriga/web/publicwebsite/PublicBlogController.java

```diff
@@ -0,0 +1,67 @@
+package edu.asu.spring.quadriga.web.publicwebsite;
+
+import java.util.ArrayList;
+import java.util.HashMap;
+import java.util.List;
+
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.context.annotation.PropertySource;
+import org.springframework.stereotype.Controller;
+import org.springframework.ui.Model;
+import org.springframework.web.bind.annotation.PathVariable;
+import org.springframework.web.bind.annotation.RequestMapping;
+import org.springframework.web.bind.annotation.RequestMethod;
+
+import edu.asu.spring.quadriga.domain.workbench.IProject;
+import edu.asu.spring.quadriga.exceptions.QuadrigaStorageException;
+import edu.asu.spring.quadriga.service.workbench.IRetrieveProjectManager;
+
+/**
+ * This controller displays the public blog contents and returns json as
+ * response with blog title, date, author and content.
+ *
+ * @author Kavinya Rajendran
+ */
+@PropertySource(value = "classpath:/user.properties")
+@Controller
+public class PublicBlogController {
+    @Autowired
+    private IRetrieveProjectManager projectManager;
+
+    /**
+     * This method gives the the projectblog
+     * 
+     * @param projectUnixName
+     *            The project unix name
+     * @param model
+     *            Model
+     * @return view
+     * @throws QuadrigaStorageException
+     */
+    @RequestMapping(value = "sites/{projectUnixName}/projectblog", method = RequestMethod.GET)
```

Comment:
Great, I'll merge the PR. But would you please change the url, so that the blog is at sites/{projectUnixName}/blog (not projectblog). Please change that as part of one of the other stories. Thanks.